download_from_csv.py

```diff
@@ -98,16 +101,35 @@ def download_finish(d):
         'logger': MyLogger(),
         'outtmpl': folder + '/' + song['name'] + ' - ' + song['artist'] + '.%(ext)s'
     }
-    url = ' '.join([song['name'], song['artist'], 'audio', 'youtube'])
+    if args.query == None:
```

Comment:
Reduce code in if-else.
Think: You are basically using `.join()` on an array. The array can change according to `args.query`